tests/test_models.py

```diff
@@ -29,3 +29,17 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+def test_daily_min_integers():
+    """Test that mean function works for an array of positive integers."""
+    from inflammation.models import daily_mean
```

Comment:
```suggestion
    from inflammation.models import daily_min
```
Importing wrong function